packages/autorest.typescript/test/rlcIntegration/lroRest.spec.ts

```diff
@@ -77,20 +80,22 @@ describe("LRO Rest Client", () => {
     });
 
     it("should handle put200Acceptedcanceled200", async () => {
-      try {
-        const initialResponse = await client
-          .path("/lro/put/200/accepted/canceled/200")
-          .put();
+      const initialResponse = await client
+        .path("/lro/put/200/accepted/canceled/200")
+        .put();
 
-        const poller = await getLongRunningPoller(client, initialResponse, {
-          intervalInMs: 0
-        });
+      const poller = await getLongRunningPoller(client, initialResponse, {
+        intervalInMs: 0
+      });
 
-        await poller.pollUntilDone();
-        assert.fail("should have thrown instead");
-      } catch (e) {
-        assert.equal(e.message, "Operation was canceled");
+      const response = await poller.pollUntilDone();
+      if (isUnexpected(response)) {
+        const error = `Unexpected status code ${response.status}`;
+        assert.fail(error);
```

Comment:
nitpick:
```suggestion
        assert.fail(`Unexpected status code ${response.status}`);
```

Reply:
We could also use `pathUnchecked` instead so we don't have to make this useless call to `isUnexpected`.